### PR TITLE
✨Source MailerLite: Correct schemas, Migrate with poetry, Make Connector Compatible with Builder

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
* Make the connector compatible with the builder

## How
* Import schema to manifest
* Update docs, 
* Completely made compatible with the builder

## User Impact
* No impact, chore for maintainability.

## Can this PR be safely reverted and rolled back?
- [x] YES 💚

### CODE REVIEW

File: airbyte-integrations/connectors/source-mailerlite/metadata.yaml [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
   githubIssueLabel: source-mailerlite
   icon: mailerlite.svg
   license: MIT
   name: MailerLite
+  releases:
+    breakingChanges:
+      1.0.0:
+        upgradeDeadline: "2024-06-30"
+        message:
+          "The verison migrates the MailerLite connector to the be compatable for connector builder.
```
Comment (girarda) [inline airbyte-integrations/connectors/source-mailerlite/metadata.yaml:24]:
```suggestion
          "The version migrates the MailerLite connector to the be compatible with the connector builder.
```

File: airbyte-integrations/connectors/source-mailerlite/metadata.yaml [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
+          Important:
+          The forms_popup stream schema from API has a breaking change to schema['properties']['settings']['properties']['schedule'],
```
Comment (girarda) [inline airbyte-integrations/connectors/source-mailerlite/metadata.yaml:26]: @btkcodedev is the breaking change related to the migration or has there been an API change?

Comment (btkcodedev) [reply airbyte-integrations/connectors/source-mailerlite/metadata.yaml:26]: API field changes only

File: docs/integrations/sources/mailerlite-migrations.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# MailerLite Migration Guide
+
+## Upgrading to 1.0.0
+
+The verison migrates the MailerLite connector to the be compatable for connector builder.
```
Comment (girarda) [inline docs/integrations/sources/mailerlite-migrations.md:5]:
```suggestion
The version migrates the MailerLite connector to the be compatable for connector builder.
```

File: docs/integrations/sources/mailerlite-migrations.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# MailerLite Migration Guide
+
+## Upgrading to 1.0.0
+
+The version migrates the MailerLite connector to the be compatable for connector builder.
```
Comment (girarda) [inline docs/integrations/sources/mailerlite-migrations.md:5]:
```suggestion
The version migrates the MailerLite connector to the be compatible for connector builder.
```

File: docs/integrations/sources/mailerlite-migrations.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# MailerLite Migration Guide
+
+## Upgrading to 1.0.0
+
+The version migrates the MailerLite connector to the be compatible for connector builder.
```
Comment (girarda) [inline docs/integrations/sources/mailerlite-migrations.md:5]:
```suggestion
The version migrates the MailerLite connector to the be compatible with connector builder.
```

**Comment by btkcodedev** [comment]
@girarda  Comments resolved and CI green
image

File: docs/integrations/sources/mailerlite.md [hunk 6]
```diff
@@ -34,6 +34,7 @@ MailerLite API has a global rate limit of 120 requests per minute.
 
 ## Changelog
 
-| Version | Date       | Pull Request                                             | Subject        |
-| :------ | :--------- | :------------------------------------------------------- | :------------- |
-| 0.1.0   | 2022-10-25 | [18336](https://github.com/airbytehq/airbyte/pull/18336) | Initial commit |
+| Version | Date       | Pull Request                                             | Subject                         |
+| :------ | :--------- | :------------------------------------------------------- | :------------------------------ |
+| 1.0.0   | 2024-05-22 | [38342](https://github.com/airbytehq/airbyte/pull/38342) | Make compatability with builder |
```
Comment (girarda) [inline docs/integrations/sources/mailerlite.md:39]:
```suggestion
| 1.0.0   | 2024-05-28 | [38342](https://github.com/airbytehq/airbyte/pull/38342) | Make compatability with builder |
```

**Review by girarda — APPROVED** [review]

**Pull request merged**